README.md

```diff
@@ -30,58 +44,22 @@ The cloudinary_android library is available in [Maven Central](http://repo1.mave
         <version>1.30.0</version>
     </dependency>
 
-
-## Try it right away
-
-Sign up for a [free account](https://cloudinary.com/users/register/free) so you can try out image transformations and seamless image delivery through CDN.
-
-*Note: Replace `demo` in all the following examples with your Cloudinary's `cloud name`.*  
-
-Accessing an uploaded image with the `sample` public ID through a CDN:
-
-    http://res.cloudinary.com/demo/image/upload/sample.jpg
-
-![Sample](https://res.cloudinary.com/demo/image/upload/w_0.4/sample.jpg "Sample")
-
-Generating a 150x100 version of the `sample` image and downloading it through a CDN:
-
-    http://res.cloudinary.com/demo/image/upload/w_150,h_100,c_fill/sample.jpg
-
-![Sample 150x100](https://res.cloudinary.com/demo/image/upload/w_150,h_100,c_fill/sample.jpg "Sample 150x100")
-
-Converting to a 150x100 PNG with rounded corners of 20 pixels:
-
-    http://res.cloudinary.com/demo/image/upload/w_150,h_100,c_fill,r_20/sample.png
-
-![Sample 150x150 Rounded PNG](https://res.cloudinary.com/demo/image/upload/w_150,h_100,c_fill,r_20/sample.png "Sample 150x150 Rounded PNG")
-
-For plenty more transformation options, see our [image transformations documentation](http://cloudinary.com/documentation/image_transformations).
-
-Generating a 120x90 thumbnail based on automatic face detection of the Facebook profile picture of Bill Clinton:
-
-    http://res.cloudinary.com/demo/image/facebook/c_thumb,g_face,h_90,w_120/billclinton.jpg
-
-![Facebook 90x120](https://res.cloudinary.com/demo/image/facebook/c_thumb,g_face,h_90,w_120/billclinton.jpg "Facebook 90x200")
-
-For more details, see our documentation for embedding [Facebook](http://cloudinary.com/documentation/facebook_profile_pictures) and [Twitter](http://cloudinary.com/documentation/twitter_profile_pictures) profile pictures.
-
-
 ## Usage
 
-### Configuration
+### Setup
 
-Each request for building a URL of a remote cloud resource must have the `cloud_name` parameter set. 
+Each request for building a URL of a remote cloud resource must have the `cloud_name` parameter set.
 Setting the `cloud_name` parameter can be done either when initializing the library, or by using the CLOUDINARY_URL meta-data property in `AndroidManifest.xml`.
 
 The entry point of the library is the `MediaManager` object. `MediaManager.init()` must be called before using the library, preferably in `Application.onCreate()`.
 Here's an example of setting the configuration parameters programmatically in your `Applicaion.onCreate(`:
-    
+
      Map config = new HashMap();
      config.put("cloud_name", "myCloudName");
      MediaManager.init(this, config);
-    
+
 Alternatively, When using the meta-data property, no configuration is required:
-    
+
     MediaManager.init(this);
 
 The added property `AndroidManifest.xml`. Note: You should only include the `cloud_name` in the value, the api secret and key should be left out of the application.
```

Comment:
Again, I know this was already there, but I don't think this readme was ever reviewed in the past. 
What does it mean to say "The added property `xxx`." as a sentence?  What are we trying to say here?

As for the note part, I suggest wording such as: 

"Only the `cloud_name` should be included.  Your API key and secret aren't necessary. 
**Note**: Your API secret should never be exposed in the application.

Reply:
The added property was referring to the meta-data you add the the file `AndroidManifest.xml` with the cloud name.
I'll change it to your suggestion as it looks better and more clear.